src/app/_components/CheckBox/CheckBox.js

```diff
@@ -0,0 +1,172 @@
+'use client';
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import cn from '@/utils/cn';
+
+export default function CheckBox({
+  value,
+  onChange,
+  name,
+  text = '',
+  subText = '',
+  disabled = false,
+  checked = false,
+  children = null,
+  extraClasses = {
```

Comment:
🟠 aren't we suppose to accept variants instead of tailwind classes directly? like in buttons PR